### PR TITLE
Refactor action

### PR DESCRIPTION
**Pull request by audio-engineer**

### CODE REVIEW

File: run_action.py [hunk 1]
```diff
@@ -30,54 +39,50 @@ def find_files_to_process(github_workspace_path=None, skip_dirs=SKIP_DIRS):
...
     os.chdir(github_workspace_path)
-    chktexrc = os.path.abspath(chktexrc_filename)
-    if os.path.exists(chktexrc):
-        return chktexrc
-    else:
-        return None
+    locale_chktexrc = os.path.abspath(".chktexrc")
 
+    if os.path.exists(locale_chktexrc):
+        return locale_chktexrc
```
Comment (j2kun) [inline run_action.py:52]: `locale` -> `local`

**Review by j2kun — APPROVED** [review]
LGTM!

**Comment by audio-engineer** [comment]
@j2kun Great! Please merge

File: run_action.py (hunk 1 above)
Comment (audio-engineer) [reply run_action.py]: Ups, autocomplete...

**Pull request merged**